File: clients/scala/src/main/scala/io/swagger/client/model/TradeBin.scala
package io.swagger.client.model

import java.util.Date



case class TradeBin (
  timestamp: Date,
  symbol: String,
  open: Double,
  high: Double,
  low: Double,
  close: Double,
  trades: Number,
  volume: Number,
  vwap: Double,
  lastSize: Number,
  turnover: Number,
  homeNotional: Double,
  foreignNotional: Double,
  id: Double)
  

File: clients/typescript-angular/API/Client/TradeBin.ts
/// <reference path="api.d.ts" />

namespace API.Client {
    'use strict';

    export interface TradeBin {

        timestamp?: date;

        symbol?: string;

        open?: number;

        high?: number;

        low?: number;

        close?: number;

        trades?: number;

        volume?: number;

        vwap?: number;

        lastSize?: number;

        turnover?: number;

        homeNotional?: number;

        foreignNotional?: number;

        id?: number;
    }

}
